README.md

```diff
@@ -144,6 +144,8 @@ The SEO Enhancer automatically adds canonical link metadata to `<head>` tags in
 <link rel="canonical" href="<SITEURL>/example.html" />
 ```
 
+If you syndicate content from third parties or include your own posts already hosted elsewhere, it is possible to explicitly assign an external canonical link for an article by adding `:canonical:` (rst) or `Canonical:` (Markdown) metadata fields to an article.
```

Comment:
Could you explain that the plugin will first check for external canonical, and if not, will fallback on `save_as`, and finally will build the URL from scratch (with `<link rel="canonical" href="<SITEURL>/example.html" />`)?

For Markdown/rst examples, you can reproduce what I did for the section "To populate the :image: field, ...".

Reply:
Better?

Reply:
Yes it is!
I just hilighted some typos.